test/integration/query-interface/schemas.test.ts

```diff
@@ -36,4 +36,28 @@ describe('QueryInterface#createSchema', async () => {
 
     expect(queryGeneratorSpy.args[0]).to.include(options);
   });
+
+  it('drops a schema', async () => {
+    await queryInterface.createSchema(testSchema);
+    const preDeletionSchemas = await queryInterface.showAllSchemas();
+    expect(preDeletionSchemas).to.include(testSchema, 'createSchema did not create testSchema');
+
+    await queryInterface.dropSchema(testSchema);
+    const postDeletionSchemas = await queryInterface.showAllSchemas();
+    expect(postDeletionSchemas).to.not.include(testSchema, 'dropSchema did not drop testSchema');
+  });
+
+  it('drops all schemas', async () => {
+    await queryInterface.dropAllSchemas();
```

Comment:
Next possible issue; `clearDatabase` (the function that runs before every test) [only drops the test schemas.](https://github.com/sequelize/sequelize/blob/main/test/support.ts#L196-L236) This drops all schemas, but doesn't create the test schema (database) afterwards and therefore MySQL/MariaDB can't select that anymore.  I think we should have a `after` for this test suite to create the test schema again so other tests are not affected. I wouldn't add it to the global since that would affect the performance of the other tests while our integration test suite is already suffering from too many DDL statements

Reply:
I think I solved it. First of all, the old `query-interface.js` doesn't have `skip` implemented properly, so I did that - as such, I passed `sequelize.config.database` to the `skip` option in the `dropAllSchemas` test, which ensures that all schemas are dropped except for the one used by other tests.